curriculum/challenges/english/01-responsive-web-design/basic-css/import-a-google-font.md

```diff
@@ -40,7 +40,11 @@ Import the `Lobster` font to your web page. Then, use an element selector to set
 You should import the `Lobster` font.
 
 ```js
-assert(new RegExp('googleapis', 'gi').test(code));
+assert(
+  $('link[href]')
+    .toArray()
+    .some((element) => new RegExp('googleapis', 'gi').test(element.href))
+);
```

Comment:
```suggestion
assert($('link[href*="googleapis" i]').length);
```
How about this?  Same basic idea, but a little more concise.

Reply:
Didn't know one could make the selector case-insensitive. Much better and nice to know. \\(^-^)/

Reply:
Neither did I until a few days ago!  I discovered it when I was trying to write some tests for the new curriculum - it's pretty useful.